scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -1822,6 +1822,10 @@ public Job loadInitalJobContent(final JobId jobId) {
             long id = jobId(jobId);
 
             Query query = session.getNamedQuery("loadJobContent").setLong("id", id);
+            if (query.uniqueResult() == null) {
+                throw new DatabaseManagerException("Invalid job id: " + jobId);
+            }
+
```

Comment:
could you catch exceptions in the following call to be robust on deserialization errors (upgrade of proactive scheduler)

Reply:
done